index.js

```diff
@@ -13,12 +14,18 @@ const port = 3000;
 // Middleware to parse JSON bodies
 app.use(express.json());
 
+const corsOptions = {
+  // Either leave it as * or change it to the URL of the frontend app.
+  // e.g.: origin: 'http://localhost:3001'
+  origin: '*',
+};
+app.use(cors(corsOptions));
```

Comment:
Necessary for use with web client. Not sure if it's a problem on iOS, but I wouldn't think so, since we're wildcarding it.

Reply:
sure, let's add it